content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -0,0 +1,155 @@
+will load. Enter the vehicle information in the vehicle form, then click :guilabel:`Save`.
+
+Vehicle form fields
+===================
+
+- :guilabel:`Model`: Select the vehicle's model from the drop-down menu. If the model is not listed,
+  type in the model name and click either :guilabel:`Create` or :guilabel:`Create and Edit`.
+- :guilabel:`License Plate`: Enter the vehicle's license plate number in this field.
+- :guilabel:`Tags`: Select any tags from the drop-down menu, or type in a new tag. There is no limit
+  on the amount of tags that can be selected.
+
+.. note::
+   The :guilabel:`Model` is the only required field on the new vehicle form. When a model is
+   selected, other fields will appear on the vehicle form, and relevant information will
+   auto-populate fields that apply to the model. If some of the fields do not appear, this may
+   indicate there is no model selected.
+
+Driver section
+--------------
+
+This section of the vehicle form relates to the person who is currently driving the car, as well as
+any plans for a change in the driver in the future, and when.
+
+- :guilabel:`Driver`: Select the driver from the drop-down menu, or type in a new driver and click
+  either :guilabel:`Create` or :guilabel:`Create and Edit`.
+- :guilabel:`Mobility Card`: If the selected driver has a mobility card listed on their employee
+  card in the *Employees* application, the mobility card number will appear in this field. If there
+  is no mobility card listed and one should be added, :doc:`edit the employee card
+  </applications/hr/employees/new_employee>` in the *Employees* application.
+- :guilabel:`Future Driver`: If the next driver for the vehicle is known, select the next driver
+  from the drop-down menu, or type in the next driver and click either :guilabel:`Create` or
+  :guilabel:`Create and Edit`.
+- :guilabel:`Plan To Change Car`: If the current driver set for this vehicle plans to change their
+  vehicle, either because they are waiting on a new vehicle that is being ordered, or this is a
+  temporary vehicle assignment and they know which vehicle they will be driving next, check this
+  box. If the current driver does not plan to change their vehicle and use this current vehicle, do
+  not check this box.
+- :guilabel:`Assignment Date`: Select the date the vehicle will be available for another driver
+  using the drop-down calendar. Select the date by navigating to the correct month and year using
+  the :guilabel:`⬅️ (left arrow)` and :guilabel:`➡️ (right arrow)` icons, then click on the specific
+  day. If this field is blank, this indicates the vehicle is currently available and can be
+  reassigned to another driver. If it is populated, the vehicle will not be available to assign to
+  another driver until the date entered.
+
+.. important::
+   A driver does *not* have to be an employee, but a driver must be listed in the *Contacts*
```

Comment:
```suggestion
   A driver does **not** have to be an employee, but a driver must be listed in the *Contacts*
```
I think anytime we are trying to emphasize a word (like not, must, cannot, etc) bold is better than italics.